blockchain/blockchain.go

```diff
@@ -90,8 +98,8 @@ func (b *Blockchain) GetBlockByHash(hash *felt.Felt) (block *core.Block, err err
 	})
 }
 
-// Store takes a block and state update and performs sanity checks before putting in the database.
-func (b *Blockchain) Store(block *core.Block, stateUpdate *core.StateUpdate) error {
+// StoreBlock takes a block and state update and performs sanity checks before putting in the database.
+func (b *Blockchain) StoreBlock(block *core.Block, stateUpdate *core.StateUpdate) error {
```

Comment:
I dont think we need to call this StoreBlock anymore right? There are no store functions anymore.

Reply:
We may still need to Store Class during `StateUpdate` though. But yeah, calling it `StoreBlock` is not needed at the moment